examples/service_account/main.tf

```diff
@@ -18,12 +18,14 @@
   Provider configuration
  *****************************************/
 provider "google" {
-  project     = "${var.service_account_project}"
+  credentials = file(var.credentials_file_path)
```

Comment:
This variable must be defined in `variables.tf`.

Reply:
@aaron-lane Thank you, I'll fix it after meeting

Reply:
@aaron-lane Done